test/unit/test_utils.js

```diff
@@ -26,6 +26,10 @@ const CMAP_PARAMS = {
   cMapPacked: true,
 };
 
+const STANDARD_FONT_DATA_URL = isNodeJS
+  ? "./external/standard_fonts/"
+  : "/build/generic/web/standard_fonts/";
```

Comment:
Shouldn't this be `"../../external/standard_fonts/"` instead here, such that the unit-tests also work when run using e.g. http://localhost:8888/test/unit/unit_test.html?

Reply:
That relative path won't work for the packaged version of the unitests since it loads relative to the worker js file. 
I've been meaning to look into this, but the unpackaged unit tests in the browser are kind of broken arleady and are loading the packaged version of the worker. With this path above they both still work.